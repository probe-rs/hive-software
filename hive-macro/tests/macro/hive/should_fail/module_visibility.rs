#[hive_macro::hive]
mod tests {}

fn main() {}
