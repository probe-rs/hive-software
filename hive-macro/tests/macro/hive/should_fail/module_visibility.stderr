error: Top level module for Hive tests must be public
 --> tests/macro/hive/should_fail/module_visibility.rs:1:1
  |
1 | #[hive_macro::hive]
  | ^^^^^^^^^^^^^^^^^^^
  |
  = help: Set the module visibility to public
  = note: example:

          #[hive]
          pub mod tests {
          // Testfunctions etc...
          }
  = note: this error originates in the attribute macro `hive_macro::hive` (in Nightly builds, run with -Z macro-backtrace for more info)
